templates/quickstart-bitbucket-dc.template.yaml

```diff
@@ -776,6 +807,9 @@ Resources:
                   - "ATL_APP_DATA_MOUNT_ENABLED=false"
                   - ""
                   - !Sub ["ATL_PRODUCT_VERSION=${BitbucketVersion}", BitbucketVersion: !Ref BitbucketVersion]
+                  - !Sub ["ATL_BB_LICENSEKEY=${LicenseKey}", LicenseKey: !If [IsLicenseKeyProvided, !Ref BitbucketLicenseKey, '']]
+                  - !Sub ["ATL_DATASET_URL=${DatasetURL}", DatasetURL: !If [IsURLProvided, !Ref BitbucketDatasetURL, '']]
+                  - !Sub ["ATL_BB_BASEURL=${BaseURL}", BaseURL: !If [UseCustomDnsName, !Sub ["${HTTP}://${CustomDNSName}", { HTTP: !If [DoSSL, https, http], CustomDNSName: !Ref CustomDnsName }], !Sub ["${HTTP}://${LoadBalancerDNSName}", { HTTP: !If [DoSSL, https, http], LoadBalancerDNSName: !GetAtt LoadBalancer.DNSName }]]]
```

Comment:
What's the difference between the `baseurl` and the `proxyName`?

Reply:
baseurl is used by bitbucket service to identify its own URL for service access purposes.

Reply:
If I understand correctly, we are using it in this call: https://bitbucket.org/atlassian/dc-deployments-automation/src/d582aa394bfd9c0ab559eec9ec844f8abc73b482/roles/bitbucket_dataset_restore/tasks/main.yml#lines-40

Why we couldn't use the proxy value? They should be the same in this case.

Reply:
I am using that value here - https://bitbucket.org/atlassian/dc-deployments-automation/src/d582aa394bfd9c0ab559eec9ec844f8abc73b482/roles/bitbucket_config/templates/bitbucket.properties.j2#lines-18.

will make the change

Reply:
I am going to revert back to original, I am not able to use the proxy value since it is not the URL but just the hostname. I need the URL to invoke the API for the subsequent tasks and also the URL for any other widgets within bitbucket

Reply:
` - !If [BitbucketAdminPassword, !Sub ["ATL_BB_ADMIN_PASSWORD='${BitbucketAdminPassword}'", BitbucketAdminPassword: !Ref BitbucketAdminPassword], !Ref "AWS::NoValue"]`